internal/pkg/agent/cmd/install.go

```diff
@@ -35,6 +35,7 @@ would like the Agent to operate.
 	}
 
 	cmd.Flags().BoolP("force", "f", false, "Force overwrite the current and do not prompt for confirmation")
+	cmd.Flags().BoolP("non-interactive", "n", false, "Install Elastic Agent in non-interactive mode")
```

Comment:
[Blockerish]
Unless this message was agreed upon, I'd ask to clarify it. The message does not clarify what _non interactve_ means. It'd be better to have the message explaining that the agent won't ask for missing parameters and will fail instead.